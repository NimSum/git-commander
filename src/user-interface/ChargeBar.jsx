import React from 'react';

export function ChargeBar (props) {
  return (
    <section className="charge-bar-container">
      <h3>Teleportation Charge:</h3>
      <div>
        <span className="charge-bar"></span>
        <span className="charge-bar"></span>
        <span className="charge-bar"></span>
      </div>
    </section>
    )
}

export default ChargeBar;