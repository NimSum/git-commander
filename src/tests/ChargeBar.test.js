import React from 'react';
import ChargeBar from '../user-interface/ChargeBar';
import Adapter from 'enzyme-adapter-react-16';
import { shallow , configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('Charge Bar' , () => {
  

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      < ChargeBar 
      />
    );
  });

  it('Should generate match charge bar snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })

})