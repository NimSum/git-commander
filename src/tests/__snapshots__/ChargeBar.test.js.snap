// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Charge Bar Should generate match charge bar snapshot 1`] = `
<section
  className="charge-bar-container"
>
  <h3>
    Teleportation Charge:
  </h3>
  <div>
    <span
      className="charge-bar"
    />
    <span
      className="charge-bar"
    />
    <span
      className="charge-bar"
    />
  </div>
</section>
`;
